mu/integration_tests/test_mcp3427.py

```diff
@@ -0,0 +1,25 @@
+from mu.sims.solar import Solar
+from mu.harness.board_sim import BoardSim
+import unittest
+import time
+
+from mu.integration_tests import int_test
+from mu.sims.bms_carrier import BmsCarrier
+
+class TestMcp3427(int_test.IntTest):
+    def setUp(self):
+        super().setUp()
+        self.board = self.manager.start(Solar, proj_name='smoke_mcp3427')
+
+    def test_mcp3427(self):
+        time.sleep(0.1)
```

Comment:
It seems like there might be some racing going on here. If I don't add this small delay, then the update store doesn't happen until after the next delay.